<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>notatka</class>
 <widget class="QWidget" name="notatka">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="przyciskZapisz">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>75</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Zapisz</string>
   </property>
  </widget>
  <widget class="QPushButton" name="przyciskDuplikuj">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>75</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Duplikuj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="przyciskZamknij">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>75</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Zamknij</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PrzyciskUsun">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>75</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Usuń</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>471</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
  <widget class="QFontComboBox" name="fontComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>162</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinWielkoscCzcionki">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::ButtonSymbols::PlusMinus</enum>
   </property>
   <property name="suffix">
    <string> px</string>
   </property>
   <property name="prefix">
    <string/>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="maximum">
    <number>128</number>
   </property>
   <property name="value">
    <number>11</number>
   </property>
  </widget>
  <widget class="QComboBox" name="comboPriorytet">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>5 - Bardzo wysoki (Pilne)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4 - Wysoki</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3 - Średni</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2 - Niski</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1 - Bardzo niski</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="checkPrzypnij">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>0</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Notatka po zaznaczeniu tej opcji zostanie przypięta i będzie zawsze na wierzchu&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Przypnij notatkę</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
